rerun_py/rerun_sdk/rerun/any_value.py

```diff
@@ -213,15 +213,15 @@ def __init__(self, drop_untyped_nones: bool = True, **kwargs: Any) -> None:
         """
         global ANY_VALUE_TYPE_REGISTRY
 
-        self.component_batches = []
+        self.component_batches = list([])
```

Comment:
nit: that's weird. Should be either `list()` or `[]` (i believe both are strictly equivalent).